pkgs/ffigen/test/native_objc_test/is_instance_test.dart

```diff
@@ -27,21 +27,21 @@ void main() {
     });
 
     test('Unrelated classes', () {
-      final base = NSObject.castFrom(BaseClass.new1(lib));
-      final unrelated = NSObject.castFrom(UnrelatedClass.new1(lib));
-      expect(BaseClass.isInstance(base), isTrue);
-      expect(BaseClass.isInstance(unrelated), isFalse);
-      expect(UnrelatedClass.isInstance(base), isFalse);
-      expect(UnrelatedClass.isInstance(unrelated), isTrue);
+      final base = NSObject.castFrom(lib, BaseClass.new1(lib));
+      final unrelated = NSObject.castFrom(lib, UnrelatedClass.new1(lib));
+      expect(BaseClass.isInstance(lib, base), isTrue);
```

Comment:
Ditto, can we get rid of the extra lib argument by making `ObjCObjectBase` abstract, and `NSObject` having a getter for library?